pages/signup.tsx

```diff
@@ -64,39 +65,51 @@ const SignUp = () => {
     ) {
       warning();
     } else if (userData.password !== userData.confirmPassword) {
-      error({ message: "Passwords do not match" });
+      error({message:"Passwords do not match"});
     } else {
-      signup(userData.email, userData.password)
-        .then(({ data }: any) => {
-          success();
-          router.push("/dashboard");
-        })
-        .catch((err: any) => {
-          error({ message: err.message });
-        });
+      signup(userData.email, userData.password).then(({ data }: any) => {
+        console.log("aa",data);
+        success();
+        router.push("/dashboard");
```

Comment:
We should first check for the data and then show `success()` and `redirect`. And please remove `console.log()` on line `71`.